3ds2sdk/build.gradle

```diff
@@ -86,6 +86,7 @@ dependencies {
     implementation "org.bouncycastle:bcprov-jdk15to18:1.69"
 
     implementation "com.nimbusds:nimbus-jose-jwt:9.21"
+    implementation 'androidx.lifecycle:lifecycle-process:2.8.4'
```

Comment:
Is this specific version of `lifecycle-process` needed?

Reply:
I think this actually isn't necessary, upon review. I think we ended up not needing it. I can't be 100% sure since I can't run the 3ds2 tests right now easily, because they're in review, but I'm going to remove this for now, and I'll bring it back if necessary.